packages/formstr-app/src/containers/MyForms/components/Local/index.tsx

```diff
@@ -56,12 +57,24 @@ const COLUMNS = [
       return <ResponsiveLink link={link} />;
     },
   },
+  {
+    key: "actions",
+    title:"Actions",
+    ellipsis: true,
+    dataIndex: "storageId",
+    width: 10,
+    render: (storageId: string, ...args: any[]) => {
+      return <div tabIndex={0} onClick={(e) => {e.stopPropagation()}}><DeleteForm type={"local"} formId={storageId} /></div>
+    }
+  }
 ];
 
 function Local() {
-  let localForms = getItem<ILocalForm[]>(LOCAL_STORAGE_KEYS.LOCAL_FORMS) ?? [];
+  let localForms = useLocalStorageItems<ILocalForm[]>(LOCAL_STORAGE_KEYS.LOCAL_FORMS) ?? [];
+  console.log(localForms)
   localForms = localForms.map((form) => ({
     ...form,
+    storageId: form.key,
```

Comment:
should storageId also be the key in this case? instead of `makeTag` ?

Reply:
key is used probably for different reasons. I did not deep dive into why key is generated using makeTag. But i needed the key used in localstorage for deleting from the localStorage.